src/main/nsis/samples/conf/process-conf.xml

```diff
@@ -36,10 +36,10 @@
             <map>
                 <entry key="sfdc.endpoint" value="https://login.salesforce.com"/> 
                 <entry key="sfdc.username" value="user@mycompany.com"/>
-                <!-- password below has been encrypted using key file, therefore it will not work without the key setting: process.encryptionKeyFile
-                the password is not a valid encrypted value, please generate the real value using encrypt.bat utility -->
+                <!-- password, specified below is invalid, please generate one entry in keystore only readable by yourself using the encrypt.bat utility -->
                 <entry key="sfdc.password" value="5555555555555555"/>
-                <entry key="process.encryptionKeyFile" value="c:\dataloader\samples\conf\sample.key"/>
+                <entry key="process.encryptionKeyFile" value="c:\users\username\sample.key"/>
```

Comment:
The keyfile should be generated automatically, at a secure default path. A comment explaining this might be good.